src/screens/CreatePost/elements/SheetAddTopic.js

```diff
@@ -169,7 +167,7 @@ const SheetAddTopic = ({refTopic, onAdd, topics, onClose, chatTopics, isAnonimit
                     <View style={{marginBottom: 5}}>
                       <Text
                         style={{
-                          color: COLORS.black,
+                          color: '#000000',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Hardcoding color values can make it difficult to maintain consistency across the app and to support features like dark mode. It's better to use a centralized color system.

```diff
- color: '#000000',
+ color: colors.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->